sunpy/net/solarnet/attrs.py

```diff
@@ -0,0 +1,56 @@
+from sunpy.net._attrs import Detector, Time, Wavelength
+from sunpy.net.attr import AttrAnd, AttrOr, AttrWalker, DataAttr, SimpleAttr
+
+
```

Comment:
```suggestion

__all__ = ["Dataset", "Limit"]

```
Should make sure we have the attrs and the client in the API documentation

Reply:
Sure i think the same 
maybe later we can add docs for  how to get data from Solarnet ?

Reply:
Yeah we will need to add that as well as the links to the reference folder in the docs folder